frontend/src/pages/CompareV2.tsx

```diff
@@ -177,3 +290,41 @@ function CompareV2(props: PageProps) {
 }
 
 export default CompareV2;
+
+/*
+if (runs) {
+        return Promise.all(
+          runIds.map(async (r, index) => {
+            const context = await getKfpV2RunContext(r);
+            const executions = await getExecutionsFromContext(context);
```

Comment:
Removing the commented logic here?

Reply:
I'm sorry I missed that, thanks for catching this comment, that was clearly supposed to be temporary - removed at https://github.com/kubeflow/pipelines/pull/7897/commits/0f319a583697fd850032b5cdfdd4a2448e45b241, thanks!